bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -351,29 +343,13 @@ var Tabzilla = (function (Tabzilla) {
     };
     Infobar.prototype.trackEvent = function (action, label, value,
                                              nonInteraction, callback) {
-        if (typeof(_gaq) !== 'object') {
-            if (callback) {
-                callback();
-            }
-
+        if (typeof(window.gaTrack) !== 'function') {
```

Comment:
Unfortunately we are unable to have in code Google Analytics and Analytics sent via Google Tag Manager live in the same page.  We decided to remove all analytics from tabzilla.js. I left the Infobar.prototype.trackEvent method in so if other sites are using the method, they won't receive an error, but no tracking will occur. I can remove the method all together if you prefer.